x/qgb/types/validator.go

```diff
@@ -97,22 +97,23 @@ func EVMAddrLessThan(e common.Address, o common.Address) bool {
 	return bytes.Compare([]byte(e.Hex())[:], []byte(o.Hex())[:]) == -1
 }
 
-// PowerDiff returns the difference in power between two bridge validator sets
-// note this is Gravity bridge power *not* Cosmos voting power. Cosmos voting
-// power is based on the absolute number of tokens in the staking pool at any
-// given time Gravity bridge power is normalized using the equation.
+// PowerDiff returns the difference in power between two bridge validator sets.
+// Note this is QGB bridge power *not* Cosmos voting power.
+// Cosmos voting power is based on the absolute number of tokens in the staking
+// pool at any given time.
+// QGB bridge power is normalized using the equation.
```

Comment:
[not blocking]

I think we should remain consistent with the rest of the repo and either wrap or if we feel the need to space things out, then we should 
```go
// sentence
//
// Note: sentence 2

Reply:
I unwrapped the comment so that its easier to read. Should i wrap it again?

Reply:
> I unwrapped the comment so that its easier to read. Should i wrap it again?

imo yeah, just so that we should probably remain consistent. In almost all, if not all, cases we wrap by line length and not sentence per standard go. we don't have a linter of it, so its not blocking, but we might want to to avoid future review feedback cycles

Reply:
I'm not super concerned about it however

Reply:
> In almost all, if not all, cases we wrap by line length and not sentence per standard go. we don't have a linter of it, so its not blocking, but we might want to to avoid future review feedback cycles

I understand, and that's  how I do it in the other docs. However, in this case, since it contains a lot of information, and is not really easy to understand, putting each sentence in a separate line made it easier to read and understand. And since we're not having any standards in the repo for comments, it made more sense to format it this way so anyone who reads it, doesn't have to separate it manually

Reply:
imo its more difficult to read lol :smile:

I'm approving, but would prefer that wrap like normal in the future. Going back and forth on silly things like this is not worth our time :slightly_smiling_face: 

I don't know of any linters that wrap comments in a specific way, and would prefer to just agree on a max comment line length and let our IDEs do the rest.

Reply:
Got you. Lets change even the docs to be more understable when writing specs. I think a lot of things could be reformulated